client/components/select-dropdown/style.scss

```diff
@@ -68,33 +71,17 @@ $compact-header-height: 35;
 	transition: background-color 0.2s ease;
 	cursor: pointer;
 
-	// down-arrow
-	&::after {
-		@include noticon('\f431', 22px);
-
-		display: block;
-		width: 20px;
-		line-height: #{ $header-height - 1 }px;
-		color: rgba( $gray, 0.5 );
-
-		position: absolute;
-			right: 14px;
-			top: 0px;
-
-		.is-compact & {
-			right: 5px;
-			line-height: #{ $compact-header-height }px;
-		}
-	}
-
-	.has-count & {
-		padding-right: 75px;
+	.gridicons-chevron-down {
+		fill: $gray-darken-20;
+		margin: 0;
+		flex-shrink: 0;
+		transition: transform 0.15s cubic-bezier( 0.175, 0.885, 0.32, 1.275 );
```

Comment:
Do we have some general guidelines for transition durations in Calypso? Nitpick: in this case `0.3s` looks slightly better to me.

Reply:
@shaunandrews what do you think about `0.2` since that would make it consistent with the foldable card animation?

Reply:
FWIW, 0.15s is already pretty consistent with other elements that use this CSS timing function. But there's no real consensus on these timings or the functions.